README.md

```diff
@@ -43,15 +44,24 @@ Before issuing an Kubertetes commands the following substitutions will be needed
 * TARGET\_HOST - This is the IP of the RH start page. It can be found in the browser by looking at the 'census-rh-performance' environment. Then 'Services & Ingress' -> ingress -> Load balancer IP. eg, 'http://http://34.107.206.101'
 * RABBITMQ\_CONNECTION - This is used if you want Locust to populate RH Firestore with test data, otherwise use 'nil'.
 
+To build and publish the docker image CATD recommended (where TAG\_NAME is set to something like "CR-123_V1"):
```

Comment:
tiny bit more information to clarify the use of TAG_NAME would be helpful.

Reply:
Explanation of tag_name purpose added. 
Please read the new text.